transport/http/outbound.go

```diff
@@ -52,6 +52,14 @@ type outbound struct {
 	URL    string
 }
 
+func (o outbound) Start() error {
+	return nil // nothing to do
```

Comment:
optional: while there's nothing to do, a good way to catch bugs (where we use an outbound without calling `Start`) might be to have this outbound verify that `Start` is called before `Call` or `Stop`